src/components/Tabs/Tabs.scss

```diff
@@ -7,6 +7,7 @@ $focus-state-box-shadow-color: rgba(92, 106, 196, 0.8);
 
 .Tabs {
   display: flex;
+  flex-wrap: wrap;
```

Comment:
@arapehl my dude - your `flex-wrap` is not playing nicely with the `fitted` variant. Poking around now but this likely needs some tweaks.

Reply:
Okay... you could just reset this within the `.fitted` selector:

```
.fitted {
  flex-wrap: nowrap;

  // ...
}
```

That seems to work fine for what I've tested... but I can't get my hands on a non-ios 13 device 😢

Reply:
Good catch! Adding `.fitted { flex-wrap: nowrap; }` fixes the issue:

| Before        | After           |
| ------------- |:-------------:|
|<img width="782" alt="Screen Shot 2019-10-04 at 10 58 32 AM" src="https://user-images.githubusercontent.com/70582/66218019-5827d880-e696-11e9-9a64-709364a73c3b.png">|<img width="787" alt="Screen Shot 2019-10-04 at 10 58 23 AM" src="https://user-images.githubusercontent.com/70582/66218018-5827d880-e696-11e9-8933-246173ec396c.png">|